src/com/cdd/bao/editor/DetailPane.java

```diff
@@ -638,11 +638,16 @@ private void recreateAssignmentFull()
 			vw.fieldName.setPrefWidth(350);
 			observeFocus(vw.fieldName, n);
 			Tooltip.install(vw.fieldName, new Tooltip("Very short label for the assignment value"));
-			
+
 			vw.dropSpec = new ComboBox<>();
-			vw.dropSpec.getItems().addAll("Include", "Include Branch", "Exclude", "Exclude Branch");
-			int sel = val.spec == Schema.Specify.WHOLEBRANCH ? 1 : val.spec == Schema.Specify.EXCLUDE ? 2 : val.spec == Schema.Specify.EXCLUDEBRANCH ? 3 : 0;
-			vw.dropSpec.getSelectionModel().select(sel);
+			vw.dropSpec.getItems().addAll(
+				Specify.ITEM.getLabel(),
+				Specify.WHOLEBRANCH.getLabel(),
+				Specify.EXCLUDE.getLabel(),
+				Specify.EXCLUDEBRANCH.getLabel(),
+				Specify.CONTAINER.getLabel()
```

Comment:
I thought about something like this but decided against it because of unpredictable ordering effects.  I understand we can impose a default ordering that works out.  But I also thought ultimately the consumer of the labels would dictate the label-ordering.